### PR TITLE
Missing tests

### PR DESCRIPTION
**Pull request by Hyperkid123**
Added missing tests for the rest of components.

### CODE REVIEW

File: src/tests/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "react/prop-types": "off"
+  }
+}
```
Comment (rvsia) [inline src/tests/.eslintrc]: Missing new line

File: src/tests/component-mapper.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (rvsia) [inline src/tests/component-mapper.test.js]: Empty line

File: src/tests/form-fields.test.js [hunk 3]
```diff
@@ -1,85 +1,243 @@
...
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
 
-  it('should render Switch with placeholder correctly', () => {
-    const wrapper = mount(
-      <SwitchField { ...props } placeholder={ 'Placeholder' } FieldProvider={ FieldProvider } />
-    );
-    expect(toJson(wrapper)).toMatchSnapshot();
-  });
+    it('should render small Switch correctly', () => {
```
Comment (rvsia) [inline src/tests/form-fields.test.js]: Small -> mini, same below

File: src/tests/form-fields.test.js [hunk 4]
```diff
@@ -1,85 +1,243 @@
...
+    });
+
+    it('should render Switch with placeholder correctly', () => {
+      const wrapper = mount(
+        <SwitchField { ...props } placeholder={ 'Placeholder' } />
+      );
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it('should render Switch with labelText correctly', () => {
```
Comment (rvsia) [inline src/tests/form-fields.test.js]: I think you can remove this test - our new switch component does not support the label text (and i think it shouldnt 😄 )

**Comment by rvsia** [comment]
@Hyperkid123 I would change the commit where are you transforming requiredLabel from function to component to fix commit to trigger a release.

File: src/tests/form-fields.test.js (hunk 4 above)
Comment (rvsia) [reply src/tests/form-fields.test.js]: (And the labelText is still being caught in formfields)

**Comment by Hyperkid123** [comment]
@rvsia sorry for the delay :hourglass:

**Pull request merged**